packages/api-specification/generate-docs.js

```diff
@@ -0,0 +1,224 @@
+const fs = require('fs');
+
+const output = JSON.parse(fs.readFileSync('type-info.json'));
+const testMatrixJson = JSON.parse(fs.readFileSync('test-report.json'));
+
+let mdObject = {};
+
+// Read as 10% or less, 25% or less etc
+// Colors are the colors for the badges on http://shields.io
+const badgeColors = {
+  10: 'red',
+  25: 'yellow',
+  50: 'yellowgreen',
+  75: 'green',
+  100: 'brightgreen'
+};
+
+const getBadgeColor = (percent) => {
+  let colorCode = '';
+  Object.keys(badgeColors).some((key) => {
```

Comment:
I see why you are using `some()` but I think it would be more readable if you used a filter or a reduce instead so that you dont need to set `colorCode` and can just use the returned value.